password/impl/src/main/java/org/wildfly/security/password/impl/SunUnixMD5CryptPasswordImpl.java

```diff
@@ -32,14 +32,13 @@
 import java.util.Arrays;
 
 import org.wildfly.common.Assert;
-import org.wildfly.security.password.spec.IteratedPasswordAlgorithmSpec;
-import org.wildfly.security.password.spec.SaltedPasswordAlgorithmSpec;
-;
 import org.wildfly.security.password.interfaces.SunUnixMD5CryptPassword;
 import org.wildfly.security.password.spec.ClearPasswordSpec;
-import org.wildfly.security.password.spec.IteratedSaltedPasswordAlgorithmSpec;
+import org.wildfly.security.password.spec.IteratedPasswordAlgorithmSpec;
 import org.wildfly.security.password.spec.IteratedSaltedHashPasswordSpec;
+import org.wildfly.security.password.spec.IteratedSaltedPasswordAlgorithmSpec;
 import org.wildfly.security.password.spec.SaltedHashPasswordSpec;
+import org.wildfly.security.password.spec.SaltedPasswordAlgorithmSpec;
```

Comment:
In addition to removing the unused semi-colon, some imports have been re-ordered here. Please remove these changes since they are not related to the issue being fixed.